apps/modernization-ui/src/design-system/table/preferences/column-preference-panel.module.scss

```diff
@@ -29,6 +30,12 @@
         gap: 1rem;
 
         @extend %thin-top;
+
+        .reset {
+            padding: 0;
+            font-size: 0.875rem;
+            min-width: auto;
```

Comment:
suggestion:  Will need to double check with the design team as to why this button is different and if that was intentional.